packages/modules/viewer-react/src/services/iModel/iModelViewerHelper.ts

```diff
@@ -69,25 +69,27 @@ export const openConnection = async (
   return;
 };
 
-export const missingViewerConnectionProps = (
-  props: Partial<RequiredViewerProps>
-): boolean => {
+export const gatherRequiredViewerProps = (
```

Comment:
@aruniverse Now relying on your `ModelLoaderProps` as Viewer input, but narrowing down to required props here. My last attempt at creating some tighter types. If you're not a fan, we can revisit when `blankConnectionProps` are removed.

Reply:
i like this a lot more